include/tdi/common/c_frontend/tdi_learn.h

```diff
@@ -0,0 +1,210 @@
+/*******************************************************************************
+ * BAREFOOT NETWORKS CONFIDENTIAL & PROPRIETARY
+ *
+ * Copyright (c) 2017-2021 Barefoot Networks, Inc.
+
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains the property of
+ * Barefoot Networks, Inc. and its suppliers, if any. The intellectual and
+ * technical concepts contained herein are proprietary to Barefoot Networks,
+ * Inc.
+ * and its suppliers and may be covered by U.S. and Foreign Patents, patents in
+ * process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material is
+ * strictly forbidden unless prior written permission is obtained from
+ * Barefoot Networks, Inc.
+ *
+ * No warranty, explicit or implicit is provided, unless granted under a
+ * written agreement with Barefoot Networks, Inc.
+ *
+ * $Id: $
+ *
+ ******************************************************************************/
+/** @file tdi_learn.h
+ *
+ *
+ *  @brief Contains TDI Learn Object APIs
+ */
+#ifndef _TDI_LEARN_H
+#define _TDI_LEARN_H
+
+#include <tdi/common/c_frontend/tdi_common.h>
+#include <tdi/common/tdi_defs.h>
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief C Learn Callback Function. This gets called upon receiving
+ * learn digests from the device.
+ * Only one cb can be associated with a Learn object at a time.
+ * If a session already has registered, then another session cannot overwrite
+ *
+ * @param[in] tdi_tgt Bf Rt target associated with the learn data
```

Comment:
Please remove "Bf Rt" or "bfrt" or "BFRT" or "BF-RT" or "BfRt" everywhere. Probably just do a search and replace over everything
Can be replaced by either TDI or tdi